tempesta_fw/ss_skb.c

```diff
@@ -734,7 +744,8 @@ ss_skb_get_room(struct sk_buff *skb, char *pspt, unsigned int len, TfwStr *it)
 int
 ss_skb_cutoff_data(const TfwStr *hdr, int skip, int tail)
 {
-	int r;
+	int r, f_len;
+	void *f_ptr;
```

Comment:
I didn't want to recalculate `f_len` for the `BUG_ON()` below. Also, we definitely need them in the case of a tail removal (see below). So this way it's uniform and easy to understand.